parcel/parcel.go

```diff
@@ -0,0 +1,101 @@
+package parcel
+
+import (
+	"database/sql"
+)
+
+const (
+	ParcelStatusRegistered = "registered"
+	ParcelStatusSent       = "sent"
+	ParcelStatusDelivered  = "delivered"
+)
+
+type Parcel struct {
+	Number    int
+	Client    int
+	Status    string
+	Address   string
+	CreatedAt string
+}
+
+type ParcelStore struct {
+	db *sql.DB
+}
+
+func NewParcelStore(db *sql.DB) ParcelStore {
+	return ParcelStore{db: db}
+}
+
+func (s ParcelStore) Add(p Parcel) (int, error) {
+	res, err := s.db.Exec("INSERT INTO parcel (client, status,address, created_At) VALUES ( :client, :status, :address, :createdAt)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("createdAt", p.CreatedAt))
+
+	if err != nil {
+		return 0, err
+	}
+
+	lastId, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(lastId), nil
+}
+
+func (s ParcelStore) Get(number int) (Parcel, error) {
+	row := s.db.QueryRow("SELECT * FROM parcel WHERE number = :number", sql.Named("number", number))
```

Comment:
Я бы явно писал столбцы, так как потом они могут добавить новый и будет ошибка
```suggestion
	row := s.db.QueryRow("SELECT number, client, status, address, created_at FROM parcel WHERE number = :number", sql.Named("number", number))
```

Reply:
Далее аналогично